content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md

```diff
@@ -0,0 +1,303 @@
+
+```kotlin
+rpcSettings {
+  port 10006
+  adminPort 10026
+}
+```
+
+### Optional configuration
+
+* `notary` &lt;config&gt; - use this configuration option to specify the node as a Notary node. **Required**> for Notary nodes. For more information, see [Notary](corda-configuration-fields.html#notary).
+
+* `devMode` &lt;boolean&gt; - use this configuration option to enable development mode when you set its value to `true`. For more information, see [devMode](corda-configuration-fields.html#devmode). For example:
+
+```kotlin
+devMode true
+```
+
+* `rpcUsers` &lt;list&gt; - use this configuration option to set the RPC users for the node. For more information, see [rpcUsers](corda-configuration-fields.html#rpcusers). You can use arbitrary values in this configuration block - "incorrect" settings will not cause a DSL error. An example follows below:
+
+```kotlin
+rpcUsers = [[ user: "user1", "password": "test", "permissions": ["StartFlow.net.corda.flows.MyFlow"]]]
+```
+
+* `configFile` &lt;string&gt; - use this configuration option to generate an extended node configuration. For example:
+
+```kotlin
+configFile = "samples/trader-demo/src/main/resources/node-b.conf"
+```
+
+* `sshdPort` &lt;integer&gt; - use this configuration option to specify the SSH port for the Docker container. This will be mapped to the same port on the host.  If `sshdPort` is specified, then that port must be available on the host and not in use by some other service. If `sshdPort` is not specified, then a default value will be used for the SSH port on the container. Use the `docker port <container_name>` command to check which port has been allocated on the host for your container. For more information, see [sshd](corda-configuration-fields.html#sshd). For example:
+
+```kotlin
+sshd {
+  port = 2222
+}
+```
+
+You can extend the `deployNodes` task with more `node {}` blocks to generate as many nodes as necessary for your application.
+
+{{< warning >}}
+When adding nodes, make sure that there are no port clashes!
+{{< /warning >}}
+
+To extend node configuration beyond the properties defined in the `deployNodes` task, use the `configFile` property with the file path (relative or absolute) set to an additional configuration file. This file should follow the standard [Node configuration](corda-configuration-file.md) format of `node.conf`. The properties set there will be appended to the generated node configuration.
+
+{{< note >}}
+If you add a property to the additional configuration file that has already been created by the `deployNodes` task, both properties will be present in generated node configuration.
+{{< /note >}}
+
+Alternatively, you can also add the path to the additional configuration file while running the gradle task via the `-PconfigFile` command-line option. However, this will result in the same configuration file being applied to all nodes.
```

Comment:
Be careful with alternatives. 

'Alternatively, you can also' could be understood as cumulative (not 'instead of' but 'in addition to'). 

More importantly, if there is more than one way of doing something, then break it out into a list that makes clear one of the following methods must be followed. 

Pls remember these are instructions, too much prose can be hard to follow. It's great for just passing on information but it makes these steps hard to follow.